qa/tasks/cbt.py

```diff
@@ -230,6 +245,7 @@ def end(self):
         testdir = misc.get_testdir(self.ctx)
         self.first_mon.run(
             args=[
+                'sudo',
```

Comment:
Why do these need to be run as root?

Reply:
it was added during the perf stat added to the test, but since i added another PR for cbt to remove the sudo from perf stat, is no longer needed. removing.